pkg/steps/release/create_release.go

```diff
@@ -247,25 +249,31 @@ func (s *assembleReleaseStep) importFromReleaseImage(ctx context.Context, dry bo
 		})
 		if err != nil {
 			if errors.IsConflict(err) {
+				log.Printf("warning: Hit update conflicts.")
 				return false, nil
 			}
 			if errors.IsForbidden(err) {
 				// the ci-operator expects to have POST /imagestreamimports in the namespace of the tag spec
 				log.Printf("warning: Unable to lock %s to an image digest pull spec, you don't have permission to access the necessary API.", s.envVar())
 				return false, nil
 			}
+			log.Printf("warning: Unknown error eccurred: %v", err)
 			return false, err
 		}
 		image := result.Status.Images[0]
```

Comment:
@hongkailiu you likely want to log the full `result` object (best as JSON ... see the dryrun logger for an example)

Reply:
Did I use the logger correctly?

Reply:
Seems [the job](https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_ci-secret-mirroring-controller/15/pull-ci-openshift-ci-secret-mirroring-controller-master-e2e-aws-build01/7) failed here:

https://github.com/openshift/ci-tools/blob/f5f120737351a4bf11bffe1ce7d685bc456e904a/pkg/steps/template.go#L377-L385

But not sure how helpful it would be to add logs there.